designcompose/src/main/java/com/android/designcompose/squoosh/SquooshRender.kt

```diff
@@ -333,12 +332,13 @@ private fun squooshTextRender(
     drawContext.canvas.save()
     drawContext.canvas.translate(0.0f, verticalCenterOffset)
 
-    val strokeWidth = style.nodeStyle.stroke.get().stroke_weight.toUniform() * density.density
+    val strokeWidth =
+        (style.nodeStyle.strokeOrNull?.strokeWeightOrNull?.toUniform() ?: 0f) * density.density
```

Comment:
I think in this case it's better to use the normal proto behavior to return a default values (which is 0) if it wasn't present, like this:
```suggestion
        (style.nodeStyle.stroke.strokeWeight.toUniform()) * density.density
```

What do you think, @rylin?

Reply:
It's a bit cleaner to use the default values, but using the `orNull` fields make it explicitly clear that those values can be null...

Reply:
Done